python/lsst/obs/decam/decamCpIsr.py

```diff
@@ -0,0 +1,145 @@
+#
+# LSST Data Management System
+# Copyright 2016 LSST Corporation.
```

Comment:
AURA/LSST, not LSSTC (cf https://github.com/lsst/templates/blob/master/CopyrightHeader.py).